minio/minio
# site replication: fix healing of bucket deletes.

This PR changes handling of bucket deletes for site replicated
setups to hold on to deleted bucket state until it syncs to all
the clusters participating in site replication.

## Description
Dependencies : needs https://github.com/minio/madmin-go/pull/98

## Motivation and Context
Proper healing of bucket deletes

## How to test this PR?
Set up site replication & perform bucket delete when a site is down
This should sync to the site when it comes back online.

## Types of changes

## Checklist:
